explorer/ExplorerProgram.ts

```diff
@@ -254,6 +254,10 @@ export class ExplorerProgram extends GridProgram {
         return columnDefs
     }
 
+    get columnDefsNotLinkedToTable(): OwidColumnDef[] {
+        return this.columnDefsByTableSlug.get(undefined) ?? []
+    }
```

Comment:
I think you know this, but:
If an explorer only has one table, then that table can be unnamed, and in that case the `tableSlug` column is also empty or nonexistent.
I guess in that case, `columnDefsByTableSlug.get(undefined)` would just return every column def??

---

If it's easier, we can also totally migrate that weirdness away, and just change all existing explorers to have named tables only, and remove all code paths that handle the special case.

Reply:
Ahh, yes, you're right. I guess it doesn't matter that much because indicator-based chart creation and csv data-based chart creation can't be mixed at the moment. But the name is def not fitting then